includes/class-newspack-newsletters-settings.php

```diff
@@ -97,6 +97,22 @@ public static function get_settings_list() {
 				'sanitize_callback' => 'sanitize_title',
 				'type'              => 'text',
 			),
+
+			/**
+			 * Letterhead Creator key
+			 *
+			 * This key is required for folks who want to integrate promotions served through
+			 * Letterhead into their newsletters. It's generally passed as a bearer token against LH
+			 * API.
+			 *
+			 * @see https://help.tryletterhead.com/hc/en-us/articles/360051316834-API-reference
+			 */
+			array(
+				'default'     => '',
+				'description' => __( 'Letterhead Key (optional)', 'newspack-newsletters' ),
+				'key'         => Newspack_Newsletters_Letterhead::LETTERHEAD_WP_OPTION_KEY,
+				'type'        => 'text',
+			),
```

Comment:
It'd be nice to add a short explanation text with a signup link. This would help discoverability and also clarify what's that setting about.

Reply:
Iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii _linked up_ "Letterhead" in my next pull, but I wasn't totally sure how I might add an additional description to this array. We have some Newspack users who are also Letterhead users already and I think that's just via some partnership and some demoing that's happened already.